libtfhe-wrapper/src/cast.rs

```diff
@@ -1,3 +1,4 @@
+#[macro_use]
```

Comment:
can probably remove these, no?

Reply:
LOL yea you added this I'll remove it in my next PR to many pushes otherwise